playkit/src/main/java/com/kaltura/playkit/player/PKSubtitlePosition.java

```diff
@@ -0,0 +1,182 @@
+package com.kaltura.playkit.player;
+
+import android.text.Layout;
+
+import com.kaltura.android.exoplayer2.text.Cue;
```

Comment:
Let's create our own `PKCue` class so that apps don't have to import ExoPlayer classes.

Reply:
We have taken the Cue variables, locally in our class.